CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@
 You shouldn't edit the node class files (all files under `diagrams/` directory) by
 yourself.
 
+## Set up your environment
+
+* cf. [DEVELOPMENT.md][DEVELOPMENT.md]
+
```

Comment:
How about this?

```md
## Setup environment

See [DEVELOPMENT](DEVELOPMENT.md) for details.
```

I think `cf.` is not proper here. And we don't have to add `.md` suffix to link title.

Reply:
I thought that using the name and linking to it was good to ensure that it's crystal clear the doc we talked about.
About `cf`, I like to use it ... maybe everywhere and maybe too much or at the wrong place :p 
I'll made the changes ^^

Reply:
done!